recipes/paragraph/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "paragraph" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
```

We have a package with the same name on the `bioconda` channel (which is a project with which conda-forge collaborates):
- https://github.com/bioconda/bioconda-recipes/blob/87d02a2ebed46dc817b1ecd6e555f84a0cee3e40/recipes/paragraph/meta.yaml
- https://anaconda.org/bioconda/paragraph/files

Please rename the package to `python-paragraph` (or with a similar prefix that seems appropriate) to avoid the name collision:
```suggestion
  name: python-{{ name|lower }}
```